src/workerd/io/worker-entrypoint.c++

```diff
@@ -558,8 +558,8 @@ kj::Promise<WorkerInterface::AlarmResult> WorkerEntrypoint::runAlarmImpl(
           // If we have an invalid timeout, set it to the default value of 30 minutes.
```

Comment:
This comment (and the line before but I cant select that in Github for some reason) need adjusting/removal